root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+/* eslint-disable no-negated-condition */
```

Comment:
I'd remove this and fix the negated conditions, unless there's a good reason to have them.

Reply:
I dont understand why there is an error, if possible please give me a hint on why the error is caused

Reply:
I gave an example here: https://github.com/metabrainz/musicbrainz-server/pull/1088/files/de7d769042e189d4b64236da28146b4235aa903c#discussion_r290369636

Instead of `!foo ? y : x`, do `foo ? x : y`.